course-definition.yml

```diff
@@ -183,11 +183,11 @@ stages:
     difficulty: hard
     description_md:
     marketing_md: |
-        In this stage, you'll connect to a peer and download a piece of the file. You'll download the piece in blocks, which you'll later combine and verify using SHA-1, a cryptographic hash value. For improved performance, you can pipeline your network requests.
```

Comment:
Context: I think this is better mentioned in the stage instructions, not the marketing copy